Simplenote/SPAppDelegate.m

```diff
@@ -315,12 +315,11 @@ - (void)logoutAndReset:(id)sender
 			
 			// hide sidebar of notelist
             [self.sidebarViewController hideSidebarWithAnimation:NO];
-			
-			[self dismissAllModalsAnimated:YES completion:^{
```

Comment:
I believe `dismissAllModals` was introduced to dismiss the spinner (and we should probably check on iPad too).

How can I reproduce the loop-y behavior? (Already tried reverting the Simperium change, and dropping this). I think we should probably either keep the `dismissAllModals`, or analyze very closely all of the things that get dismissed by this call.